.github/actions/linter_tests/action.yaml

```diff
@@ -38,6 +38,10 @@ runs:
       shell: bash
       working-directory: ${{ inputs.path }}
 
+    - name: Install packages
```

Comment:
```suggestion
    - name: Install packages
      if: env.RUNNER_OS == 'Linux'
```

Reply:
latest-macos supports Perl so maybe it is better to add it. I use MacOS M1 as my main platform so I don't want to disable tests for this OS.

Reply:
Hi thanks for addressing the comments. We can run this test on Mac, it was just missing the correct installation step. Assuming this next CI run passes I'll do a final pass and stamp